### PR TITLE
Store 2 sets of snapshots for 3d scans

### LINKED ISSUE #134: GDA: Take two snapshots
To expedite https://github.com/DiamondLightSource/python-artemis/issues/91 it would be easier to take the two snapshots in GDA for now and then work on adding more to Artemis later.

# Acceptance Criteria
* GDA takes two snapshots with a 90 deg rotation between them
* GDA passes the path for both snaphots to Artemis
* Artemis correctly puts these into ISpyB and they are correctly shown on synchweb

### PR DESCRIPTION
**Pull request by bentom08**
Fixes #134

To Test -

Artemis side should just need unit tests to pass, GDA side of this issue will need beamline testing

### CODE REVIEW

**Comment by DominicOram** [comment]
See https://gerrit.diamond.ac.uk/c/gda/gda-mx/+/35058/11..12 for GDA side

**Comment by bentom08** [comment]
https://gerrit.diamond.ac.uk/c/gda/gda-mx/+/35058/11..13 new GDA patchset for this change

File: src/artemis/ispyb/ispyb_dataclass.py [hunk 1]
```diff
@@ -29,7 +29,8 @@ class IspybParams:
     )
 
     synchrotron_mode: str
-    xtal_snapshots: List[str]
+    xtal_snapshots_y: List[str]
+    xtal_snapshots_z: List[str]
```
Comment (DominicOram) [inline src/artemis/ispyb/ispyb_dataclass.py:33]: **nit**: It's only y/z 'cos we're assuming 0/90. Maybe better to do something like omega start and omega end?

**Review by DominicOram — CHANGES_REQUESTED** [review]
Looks good generally, need to fix the issue in GDA

**Review by DominicOram — APPROVED** [review]
Great, thank you!

**Pull request merged**